nixos/modules/services/monitoring/prometheus/exporters/redis.nix

```diff
@@ -9,6 +9,7 @@ in
   port = 9121;
   serviceOpts = {
     serviceConfig = {
+      RestrictAddressFamilies = [ "AF_INET" "AF_INET6" "AF_UNIX" ];
```

Comment:
This will render to
`[ "AF_INET" "AF_INET6" "AF_UNIX" "AF_INET" "AF_INET6" ]` due to merging with the default set from all exporters. You should just add `"AF_UNIX"`

Reply:
Oki, doing that then :+1: